lib/matplotlib/collections.py

```diff
@@ -1171,29 +1171,52 @@ def __init__(self, segments,     # Can be None.
                  **kwargs
                  ):
         """
-        *segments*
-            a sequence of (*line0*, *line1*, *line2*), where::
+        Parameters
+        ----------
+        segments :
+            A sequence of (*line0*, *line1*, *line2*), where::
 
                 linen = (x0, y0), (x1, y1), ... (xm, ym)
 
             or the equivalent numpy array with two columns. Each line
             can be a different length.
 
-        *colors*
-            must be a sequence of RGBA tuples (e.g., arbitrary color
+        colors : sequence, optional
+            A sequence of RGBA tuples (e.g., arbitrary color
             strings, etc, not allowed).
 
-        *antialiaseds*
-            must be a sequence of ones or zeros
+        antialiaseds : sequence, optional
+            A sequence of ones or zeros.
 
-        *linestyles* [ 'solid' | 'dashed' | 'dashdot' | 'dotted' ]
-            a string or dash tuple. The dash tuple is::
+        linestyles : string, tuple, optional
+            Either one of [ 'solid' | 'dashed' | 'dashdot' | 'dotted' ], or
+            a dash tuple. The dash tuple is::
```

Comment:
I wonder if the short names are also fine (I mean "-", "--", "-.", ":")? (And possibly some other exotic options sometimes accepted by some methods and classes related to lines)